class CustomException(Exception):
    pass

raise CustomException("Это пользовательское исключение.")
